sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -255,3 +287,43 @@ case class AlterViewAsCommand(
     session.sessionState.catalog.alterTable(updatedViewMeta)
   }
 }
+
+/**
+ * Drops a view which matches the given name.
+ *
+ * If the view name contains database prefix, this command will drop a permanent view matching the
+ * given name, or throw an exception if view not exist. Else, this command will try to drop a local
+ * temporary view first, if view not exist, try global temporary view next, if still not exist, try
```

Comment:
If that is a design choice, never mind.
I think it's okay if this PR can provide `DROP GLOBAL VIEW` together.